app/src/main/java/com/douglasbruce/tasky/core/designsystem/component/Buttons.kt

```diff
@@ -0,0 +1,38 @@
+package com.douglasbruce.tasky.core.designsystem.component
+
+import androidx.annotation.StringRes
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun TaskyButton(
+    @StringRes text: Int,
```

Comment:
Here, I'd recommend to pass in a string instead, that's how composables do that typically. It has the advantage that you are more flexible with passing strings. You can either pass `stringResource(yourId)` or a dynamic string

Reply:
Addressed within PR - https://github.com/DouglasBruce/Tasky/pull/5